### DISCUSSION TITLE
OTA Firmware Update for sonoff ZBMINIL2

### DISCUSSION THREAD

**Discussion by davidcreager** [Koenkk/zigbee2mqtt#19782] [Devices]
Hi,
Apologies if this is a 'stupid' question.
I have a number of zbmiml2 devices and they all appear as OTA available.   However, selecting update just sits there with n/a appearing in time available.  The current firmware is 4106 and the available is 4110.   I am running zigbee2mqtt in docker

![image](https://github.com/Koenkk/zigbee2mqtt/assets/17993498/94640ee0-9c17-4032-ba18-2682eb7913e9)

**Comment by qfina** [comment]
I have the same, updates started, albeit super slow, but never ended.  I had to remove and re-add the device to stop the 'progress'.
